src/transforms/v2-to-v3/__fixtures__/api-promise-class-member.input.js

```diff
@@ -0,0 +1,18 @@
+import * as AWS from 'aws-sdk';
+
+export class SecretsManager {
+  constructor(client = new AWS.SecretsManager()) {
+    this.client = client;
+  }
+
+  async getSecretValue(secretId) {
+    const response = await this.client
+      .getSecretValue({ SecretId: secretId })
+      .promise();
```

Comment:
does it work on this case?
```
const x = client.method();
const result = x.promise();
```

Reply:
I don't think it would.
It does work for promise called on variable declarator tested in https://github.com/awslabs/aws-sdk-js-codemod/blob/main/src/transforms/v2-to-v3/__fixtures__/api-promise-variable-declarator.input.js

I'll test and add a new feature request.

Reply:
Feature request created at https://github.com/awslabs/aws-sdk-js-codemod/issues/185